src/db/migrations/1626689144415-HospitalsMigration.ts

```diff
@@ -0,0 +1,16 @@
+import {MigrationInterface, QueryRunner} from "typeorm";
+
+export class HospitalsMigration1626689144415 implements MigrationInterface {
+    name = 'HospitalsMigration1626689144415'
+
+    public async up(queryRunner: QueryRunner): Promise<void> {
+        await queryRunner.query(`ALTER TABLE "hospitals" DROP COLUMN "phoneNumbers"`);
```

Comment:
as this migration were not merged, you can regenerate migration with updates. only if this one were pushed/deployed you must use update migrations